aliases/available/vim.aliases.bash

```diff
@@ -2,20 +2,14 @@
 cite 'about-alias'
 about-alias 'vim abbreviations'
 
-if _command_exists vim; then
-	alias v='$VIM'
-	# open the vim help in fullscreen incorporated from
-	# https://stackoverflow.com/a/4687513
-	alias vimh='${VIM} -c ":h | only"'
-fi
+_command_exists vim || return
+
+alias v='${VIM:-vim}'
+# open the vim help in fullscreen incorporated from
+# https://stackoverflow.com/a/4687513
+alias vimh='${VIM:-vim} -c ":h | only"'
```

Comment:
I kind of dig the `"${VIM:-vim}"` usage, **but** in the presence of `[ -n "${VIM}" ]` I don't think its fair to assume that `"${VIM}"` refers specifically to `vim` (i.e `"${VIM}"` may not refer to `vim` at all but something compatible that the user still wants aliases for)

I think the check/bail should be something like:
```
[ -n "${VIM}" ] || _command_exists vim || return
```

Additionally, assuming the bail check passes,  if `vim` doesn't exist, it might be nice to do the user a favor and create an alias for it:
```
_command_exists vim || alias vim='${VIM}'
```

Reply:
OK I see now that the usage of `$VIM` here is a bug introduced in #1938 :
* [changes to vim.aliases.bash](https://github.com/Bash-it/bash-it/commit/757a5bf25bc6135b9d66082bbf5d86c3cac27f58#diff-e7fed7396da8295a60f92e4862ec9c19eb2ac4fb86a7690f81468dac581740dc)

That commit removes the *creation* of `$VIM`:
```
< VIM=$(command -v vim)
```

**BUT** it didn't remove the *usages* of `$VIM`

So it a minimum just replace all usages of `$VIM` with `vim` and then we can re-valuate from there ...

#Keep_PRs_Small_And_Focused